sos/collector/clusters/juju.py

```diff
@@ -161,12 +161,13 @@ def _get_model_info(self, model_name):
     def _get_juju_version(self):
         """Grab the version of juju"""
         res = sos_get_command_output("juju version")
-        return res['output'].split("-")[0]
+        return res['output']
```

Comment:
I think this split was done to workaround `parse_version` as the juju version would look like `2.9.46-ubuntu-amd64`.